### PR TITLE
refactor: libdave camel massacre 🐪🔪

### PR DESCRIPTION
**Pull request by braindigitalis**
Finally updates libdave to get rid of `camelCase_` members with underscore on the end. This refactor updates:

* Public and private variables that are part of a struct or class
* Public and private class function names
* Function name parameters

Where needed, a little reformatting.

TODO at a later date: Various parts of libdave mix and match uint64_t and std::string for snowflake storage. standardize them all on D++ `dpp::snowflake` where possible.

## Code change checklist

- [x] I have ensured that all methods and functions are **fully documented** using doxygen style comments.
- [x] My code follows the [coding style guide](https://dpp.dev/coding-standards.html).
- [x] I tested that my change works before raising the PR.
- [x] I have ensured that I did not break any existing API calls.
- [x] I have not built my pull request using AI, a static analysis tool or similar without any human oversight.

### CODE REVIEW

**Comment by ruslan-ilesik** [comment]
🐍🔪🐪 => DPP

File: src/dpp/dave/codec_utils.cpp [hunk 1]
```diff
@@ -31,98 +31,95 @@
...
 		// if we're starting a new byte
 		// check if this is an emulation prevention byte
 		// which we skip over
-		if (bitIndex == 0) {
-			if (byteIndex >= 2 && payloadByte == kEmulationPreventionByte &&
-				payload[byteIndex - 1] == 0 && payload[byteIndex - 2] == 0) {
-				payloadBitIndex += 8;
+		if (bit_index == 0) {
+			if (byte_index >= 2 && payload_byte == emulation_prevention_byte &&
+			    payload[byte_index - 1] == 0 && payload[byte_index - 2] == 0) {
```
Comment (Jaskowicz1) [inline src/dpp/dave/codec_utils.cpp:57]: Space indentation

File: src/dpp/dave/codec_utils.cpp [hunk 2]
```diff
@@ -31,98 +31,95 @@
...
-			i += kH26XNaluShortStartSequenceSize;
-		}
-		else if (buffer[i + 2] == kH26XStartCodeEndByteValue) {
+			i += nalu_short_start_sequence_size;
+		} else if (buffer[i + 2] == start_code_end_byte_value) {
 			// third byte matches the start code end byte, might be a start code sequence
-			if (buffer[i + 1] == kH26XStartCodeLeadingBytesValue &&
-				buffer[i] == kH26XStartCodeLeadingBytesValue) {
+			if (buffer[i + 1] == start_code_leading_bytes_value &&
+			    buffer[i] == start_code_leading_bytes_value) {
```
Comment (Jaskowicz1) [inline src/dpp/dave/codec_utils.cpp:107]: Ditto

File: src/dpp/dave/cryptor_manager.cpp [hunk 3]
```diff
@@ -44,58 +44,53 @@ key_generation compute_wrapped_generation(key_generation oldest, key_generation
...
+	if (!newest_processed_nonce) {
 		return true;
 	}
 
-	auto bigNonce = compute_wrapped_big_nonce(generation, nonce);
-	return bigNonce > *newestProcessedNonce_ ||
-	  std::find(missingNonces_.rbegin(), missingNonces_.rend(), bigNonce) != missingNonces_.rend();
+	auto wrapped_big_nonce = compute_wrapped_big_nonce(generation, nonce);
+	return wrapped_big_nonce > *newest_processed_nonce ||
+	       std::find(missing_nonces.rbegin(), missing_nonces.rend(), wrapped_big_nonce) != missing_nonces.rend();
```
Comment (Jaskowicz1) [inline src/dpp/dave/cryptor_manager.cpp:64]: Ditto

File: src/dpp/dave/decryptor.cpp [hunk 4]
```diff
@@ -34,56 +34,52 @@ using namespace std::chrono_literals;
...
+	scope_exit cleanup([&] { return_frame_processor(std::move(local_frame)); });
 
 	// Skip decrypting for silence frames
-	if (mediaType == media_audio && encryptedFrame.size() == OPUS_SILENCE_PACKET.size() &&
-	std::memcmp(encryptedFrame.data(), OPUS_SILENCE_PACKET.data(), OPUS_SILENCE_PACKET.size()) == 0) {
-		creator.log(dpp::ll_trace, "decrypt skipping silence of size: " + std::to_string(encryptedFrame.size()));
-		if (encryptedFrame.data() != frame.data()) {
-			std::memcpy(frame.data(), encryptedFrame.data(), encryptedFrame.size());
+	if (this_media_type == media_audio && encrypted_frame.size() == OPUS_SILENCE_PACKET.size() &&
+	    std::memcmp(encrypted_frame.data(), OPUS_SILENCE_PACKET.data(), OPUS_SILENCE_PACKET.size()) == 0) {
```
Comment (Jaskowicz1) [inline src/dpp/dave/decryptor.cpp:77]: Space indent

File: src/dpp/dave/decryptor.cpp [hunk 5]
```diff
@@ -92,81 +88,74 @@ size_t decryptor::decrypt(media_type mediaType,
...
 	else {
-		stats_[mediaType].decrypt_failure++;
-		creator.log(dpp::ll_warning, "decrypt failed, no valid cryptor found, type: " + std::string(mediaType ? "video" : "audio") +
-			", encrypted frame size: " + std::to_string(encryptedFrame.size()) +
-			", plaintext frame size: " + std::to_string(frame.size()) +
-			", number of cryptor managers: " + std::to_string(cryptorManagers_.size()) +
-			", pass through enabled: " + std::string(canUsePassThrough ? "yes" : "no")
+		stats[this_media_type].decrypt_failure++;
+		creator.log(dpp::ll_warning, "decrypt failed, no valid cryptor found, type: " + std::string(this_media_type ? "video" : "audio") +
+					     ", encrypted frame size: " + std::to_string(encrypted_frame.size()) +
```
Comment (Jaskowicz1) [inline src/dpp/dave/decryptor.cpp:129]: 2 tabs 4 spaces here (and the lines below)

File: src/dpp/dave/version.h [hunk 6]
```diff
@@ -44,5 +44,4 @@ using signature_version = uint8_t;
  */
 protocol_version max_protocol_version();
 
-} // namespace dpp::dave
-
+}
```
Comment (Jaskowicz1) [inline src/dpp/dave/version.h:47]: New line needed after.

**Review by Jaskowicz1 — CHANGES_REQUESTED** [review]

**Review by Jaskowicz1 — APPROVED** [review]
lgtm! Thank ya for getting rid of camelCase

**Pull request merged**